README.md

```diff
@@ -2,15 +2,15 @@
 
 # Fairlearn
 
-Fairlearn is a Python package that empowers developers of artificial intelligence (AI) systems to assess their system's fairness and mitigate any observed unfairness issues. Fairlearn contains mitigation algorithms as well as a Jupyter widget for model assessment. Besides the source code, this repository also contains Jupyter notebooks with examples of Fairlearn usage.
+Fairlearn is a Python package that empowers developers of artificial intelligence (AI) systems to assess their system's fairness and mitigate any observed unfairness issues. Fairlearn contains mitigation algorithms as well as metrics for model assessment. Besides the source code, this repository also contains Jupyter notebooks with examples of Fairlearn usage.
 
 Website: https://fairlearn.github.io/
 
 - [Current release](#current-release)
 - [What we mean by _fairness_](#what-we-mean-by-fairness)
 - [Overview of Fairlearn](#overview-of-fairlearn)
   - [Fairlearn algorithms](#fairlearn-algorithms)
-  - [Fairlearn dashboard](#fairlearn-dashboard)
+  - [Fairlearn metrics](#fairlearn-metrics)
```

Comment:
I'd prefer to put metrics before algorithms throughout the README.

Reply:
Actually, that's a good point! Not sure why it wasn't like that to begin with!

Reply:
Done!